Dockerfile

```diff
@@ -9,9 +9,6 @@ RUN echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/a
 RUN apt-get update
 RUN apt-get install -y kubectl
 
-# Install testkube plugin
-RUN curl -sSLf https://kubeshop.github.io/testkube/install.sh | bash
-
```

Comment:
It is removed from the dockerfile, since it is not possible to send input variables at this point.